openqa-trigger-bisect-jobs

```diff
@@ -111,7 +111,8 @@ client_args = [
 def call(cmds, dry_run=False):
     log.debug("call: %s" % cmds)
     return subprocess.check_output(
-        (["echo", "Simulating: "] if dry_run else []) + cmds
+        (["echo", "Simulating: "] if dry_run else []) + cmds,
+        stderr=subprocess.STDOUT, shell=True
```

Comment:
Why do we need that?

Reply:
I read that this ensures capturing the error and display it, especially when you redirect the the stderr. the problem I had, from the old PR was how to catch some messages on the test.

Reply:
as I understand it, yes. but it might not be mandatory. the diff is that it involves `sh` rather than direct execution and it looks useful when you run shell commands. I dont know the exact impact though.

Reply:
We shouldn't use something just because it looks useful, but we don't know what it does :)

https://docs.python.org/3/library/subprocess.html#frequently-used-arguments

> If passing a single string, either `shell` must be `[True]`(https://docs.python.org/3/library/constants.html#True) (see below) or else the string must simply name the program to be executed without specifying any arguments.

We are not executing any shell commands in this script, we are only doing system calls. We are passing the program and its arguments as an array, not a single string.

The shell is needed for example when we do something like `command | grep`.

Reply:
I remove it. I have some questions but we can discuss it out of here for less noise. thanks